torch/_meta_registrations.py

```diff
@@ -5545,8 +5544,8 @@ def meta__scaled_dot_product_flash_attention(
         None,
         max_seqlen_batch_q,
         max_seqlen_batch_k,
-        torch.empty((), dtype=torch.long, device="meta"),
-        torch.empty((), dtype=torch.long, device="meta"),
+        torch.empty((2), dtype=torch.uint64, device="meta"),
+        torch.empty((), dtype=torch.uint64, device="meta"),
```

Comment:
these relate to the philox seed / offset, right? dtype / shape was wrong before?

Reply:
Exactly right